extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -533,6 +454,55 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 
 	public async onPageEnter(): Promise<void> {
 		await this.getSubscriptionValues();
+		this.wizard.registerNavigationValidator((pageChangeInfo) => {
```

Comment:
Hmm, I don't really like the pattern of "when the user clicks next we check if things are valid and give them an error if not". Having the button only be enabled through the normal page validation logic should be the preferred way of doing things. Is this the possible bug you were telling me about?

Reply:
I have created a new PR for that. 

Moreover, I don't think it is a good UX when the user enter the page and is thrown a bunch of errors because they have not select proper options in the dropdown.

Reply:
Closing this since we'll follow up with design separately on the correct way to handle validation errors.